guides/ruby-gems/gemfury.md

```diff
@@ -0,0 +1,50 @@
+# Gemfury usage
+
+Alternative way to use private gems on multiple projects and add them to dependencies is publishing on hosted repository [gemfury](https://manage.fury.io/dashboard/dvelp).
+
+## Gem publishing
+
+To publish new gem or new version of the gem you need to do following steps:
```

Comment:
To publish new gem or new version of the gem you need to follow the steps below:

- Install gemfury gem: `gem install gemfury`
- Authenticate with your Gemfury login credentials:  `fury login`
- Build the gem: `gem build <your_gem_name>.gemspec`
- Upload your package: `fury push <your_gem_name>-<your_gem_version>.gem --as=dvelp`

That’s it!